code/modules/mob/living/simple_animal/abnormality/aleph/army_in_black.dm

```diff
@@ -0,0 +1,498 @@
+	return
+
+/mob/living/simple_animal/hostile/army_enemy/Life()
+	. = ..()
+	if(!.)
+		return
+	if(targetted_beacon)
+		GoToBeacon()
+	return
+
+/mob/living/simple_animal/hostile/army_enemy/Initialize()
+	..()
+	addtimer(CALLBACK(src, .proc/Explode), 120 SECONDS)
+	var/list/depts = shuffle(GLOB.department_centers)
+	if(!LAZYLEN(depts))
+		return
+	for (var/turf/T in depts)
+		if(!(get_area(T) in (get_areas_in_range(20, get_area(src)))))//if they're not within 20 tiles of T, pick the next one.
+			continue//FIXME: the final element should forcibly attempt to spawn these
+		if(/obj/effect/pink_beacon in T.contents)//don't spawn 2 in the same dept(FIXME: doesn't work, even with sleep() called on SpawnAdds!)
+			continue
+		var/obj/effect/pink_beacon/P = new(get_turf(T))//beacon
+		targetted_beacon = P
+		P.targetted_army = src
+		INVOKE_ASYNC(src, .proc/SetSpeed)
+		break
+
+/mob/living/simple_animal/hostile/army_enemy/death()
+	animate(src, alpha = 0, time = 10 SECONDS)
+	QDEL_IN(src, 10 SECONDS)
+	..()
+
+/mob/living/simple_animal/hostile/army_enemy/proc/SetSpeed()
+	var/dist_travelled = get_path_to(src, get_turf(targetted_beacon), /turf/proc/Distance_cardinal, 0, 200)
+	move_to_delay -= clamp((LAZYLEN(dist_travelled) / 4), 0, 15)//armies that spawn closest to dept centers can actually be suppressed this way, while further ones remain a threat. Math needs tweaking
+
+/mob/living/simple_animal/hostile/army_enemy/proc/FearEffect()
+	for(var/mob/living/carbon/human/H in view(7, src))
+		if(H in fear_affected)
+			continue
+		if(HAS_TRAIT(H, TRAIT_COMBATFEAR_IMMUNE))
+			continue
+		H.adjustSanityLoss(H.maxSanity*(-0.3))
+		if(H.sanity_lost)
+			continue
+		to_chat(H, "<span class='warning'>Oh dear.</span>")
+	return
+
+//explosion definition
+/mob/living/simple_animal/hostile/army_enemy/proc/Explode()
+	if(QDELETED(src))
+		return
+	playsound(get_turf(src), 'sound/abnormalities/armyinblack/black_explosion.ogg', 125, 0, 8)
+	visible_message("<span class='danger'>[src] suddenly explodes!</span>")
+	for(var/mob/living/simple_animal/hostile/abnormality/P in range(boom_radius, src))
+		P.datum_reference.qliphoth_change(-1)
+	for(var/mob/living/carbon/human/H in view(boom_radius, src))
+		H.apply_damage(boom_damage, WHITE_DAMAGE, null, H.run_armor_check(null, WHITE_DAMAGE), spread_damage = TRUE)
+	new /obj/effect/temp_visual/black_explosion(get_turf(src))
+	qdel(src)
+
+//aoe attacks
+/mob/living/simple_animal/hostile/army_enemy/ListTargets()//Move Move Move! This prevents it from aggroing anything
+	return FALSE
+
+/mob/living/simple_animal/hostile/army_enemy/attackby(obj/item/W, mob/user, params)
+	. = ..()
+	OpenFire(user)
+
+/mob/living/simple_animal/hostile/army_enemy/OpenFire()
+	if(shot_cooldown <= world.time)
+		BlackAoe()
+
+/mob/living/simple_animal/hostile/army_enemy/proc/BlackAoe()
+	if(stat == DEAD)
+		return
+	for(var/turf/T in view(4, src))
+		//FIXME: just make them black
+		new /obj/effect/temp_visual/army_hearts(get_turf(T))
```

Comment:
```suggestion
		var/obj/effect/temp_visual/army_hearts/AH = new (get_turf(T))
		AH.color = COLOR_BLACK
```
Or just make a subtype that's black.

Reply:
That's an old comment I forgot to get rid of, they use separate effects that have unique icons